From rain_maker@root-forum.org  Fri Sep 11 15:24:42 2009
From: "Axel K." <rain_maker@root-forum.org>
Date: Thu, 3 Sep 2009 21:53:37 +0200
Subject: Staging: rt3090: rename device from raX to wlanX
To: Bartlomiej Zolnierkiewicz <bzolnier@gmail.com>
Cc: Greg KH <gregkh@suse.de>
Message-ID: <200909032153.39066.rain_maker@root-forum.org>
Content-Disposition: inline


This patch sets "wlan" as the default suffix for naming the device, a
change which has also been previously applied to rt2860/rt2870 in
staging.

Signed-off-by: Axel Koellhofer <rain_maker@root-forum.org>
Acked-by: Bartlomiej Zolnierkiewicz <bzolnier@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/rt3090/rtmp_def.h |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/staging/rt3090/rtmp_def.h
+++ b/drivers/staging/rt3090/rtmp_def.h
@@ -1414,8 +1414,8 @@
 #define INT_APCLI			0x0400
 #define INT_MESH			0x0500
 
-#define INF_MAIN_DEV_NAME		"ra"
-#define INF_MBSSID_DEV_NAME		"ra"
+#define INF_MAIN_DEV_NAME		"wlan"
+#define INF_MBSSID_DEV_NAME		"wlan"
 #define INF_WDS_DEV_NAME		"wds"
 #define INF_APCLI_DEV_NAME		"apcli"
 #define INF_MESH_DEV_NAME		"mesh"
